test/image.test.js

```diff
@@ -83,6 +83,24 @@ describe('Image', function () {
     return assertRejects(loadImage(`${png_clock}fail`), Error)
   })
 
+  it('returns a nice, coded error for fopen failures', function (done) {
+    const img = new Image()
+    img.onerror = err => {
+      assert.equal(err.code, 'ENOENT')
```

Comment:
maybe assert `.syscall` and `.path` here as well?

Reply:
👍 added